README.md

```diff
@@ -30,17 +31,17 @@ Usage
 -----
 
 There's no need to configure anything to run the application. If you have
-installed the [Symfony client][4] binary, run this command to run the built-in
-web server and access the application in your browser at <http://localhost:8000>:
+[installed Symfony][4], run this command to run the built-in web server and
+access the application in your browser at <http://localhost:8000>:
```

Comment:
It can be another port iirc
Maybe just write « access the app on the url displayed by the command » ?

Reply:
Or even better?
> access the app on the URL displayed by the command (`https://localhost:8000` by default)

Something like this